packages/arb-util/hashing/hashing.go

```diff
@@ -46,6 +46,14 @@ func Address(input common.Address) []byte {
 	return solsha3.Address(input.ToEthAddress())
 }
 
+func AddressArray(input []common.Address) []byte {
+	addresses := make([]ethcommon.Address, 0, len(input))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)